app/renderer/components/navigation/navigationBar.js

```diff
@@ -83,17 +83,7 @@ class NavigationBar extends React.Component {
 
   onStop () {
     ipc.emit(messages.SHORTCUT_ACTIVE_FRAME_STOP)
```

Comment:
I think we should put this ipc message in there as well. When we get rid of it, we'll just switch over to the browser process handling WINDOW_ON_STOP